src/test/java/step3/RandomNumberGeneratorTest.java

```diff
@@ -0,0 +1,17 @@
+package step3;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RandomNumberGeneratorTest {
+
+    @DisplayName("RandomNumberGenerator 로 생성되는 숫자 범위 테스트")
+    @RepeatedTest(10)
+    void generateNumber() {
+        NumberGenerator numberGenerator = new RandomNumberGenerator();
+        assertThat(numberGenerator.generateNumber())
+                .isBetween(0, 9);
```

Comment:
```suggestion
                .isBetween(0, 9);
```

만약, `private static final int MAX = 10;` 값이 `5` 으로 변경된다면 깨지기 쉬운 테스트가 되지 않을까요?

`private`인 만큼 클래스 내부 세부사항이라 생각되어요. 그렇다는건 외부에서 그러한 조건을 테스트를 한다는건 추측의 범위가 될 수 있다고 생각해요.

어떻게 생각하시나요? 🤔